File: beginner/c/1101.c
#include <stdio.h>

int main() {

    int n, i, m, soma = 0, menor, maior;

    for(;;){
        soma = 0;
        scanf("%d %d", &n, &m);

        if(n <= 0 || m <= 0)
            break;
        else{
            if(n < m){
                menor = n;
                maior = m;
            } else{
                menor = m;
                maior = n;
            }

            for(i = menor; i <= maior; i++){
                printf("%d ", i);
                soma += i;
            }
            printf("Sum=%d\n", soma);
        }
    }

    return 0;
}

File: beginner/python/1101.py
while True:
    soma = 0
    n, m = map(int, input().split())

    if n <= 0 or m <= 0:
        break
    else:
        menor = n if n < m else m
        maior = m if n < m else n

        for i in range(menor, maior + 1):
            print(i, end=' ')
            soma += i

        print("Sum=%d" % soma)
